config/initializers/forest_liana.rb

```diff
@@ -1,2 +0,0 @@
-ForestLiana.env_secret = Rails.application.secrets.forest_env_secret
```

Comment:
This file needs to be deleted, not just the contents removed.  I can't tell if it has been removed, or just emptied out.

Reply:
ok I did not see this